### PR TITLE
slowed clock to 80Mhz

### PR DESCRIPTION
**Pull request by samuk**
slowed clock to 80Mhz

### CODE REVIEW

**Comment by samuk** [comment]
Tested with BLE and WiFi versions. 

Page load is a little slow on WiFi but it still works.

**Comment by paidforby** [comment]
Thanks for the contribution! Sorry about not reviewing sooner, I've been busy the last week and most of my disaster-radio energy has been dedicated to addressing the String/struct issue. This contribution makes sense to me. Though I'd rather keep the clock speed at the default (240Mhz I believe) and just have these line in there so people know there is the option. Eventually, we could add clock speed to the user preferences as something that could be set on reboot or maybe this could be part of a pre-compiled "low-power" binary that enables other features that sacrifice performance for energy usage.

I'm going to add clock speed as a define in `config.h`, then I'll merge this commit.

File: firmware/esp32/main.ino [hunk 1]
```diff
@@ -387,6 +387,9 @@ void setup()
 {
   Serial.begin(115200);
   delay(200);
+  
+  setCpuFrequencyMhz(80); //Set CPU clock to 80MHz fo example
```
Comment (paidforby) [inline firmware/esp32/main.ino]: I'll change to `setCpuFrequencyMhz(CLOCK_SPEED)` and then in `config.h` add
`#define CLOCK_SPEED 240`

**Comment by paidforby** [comment]
Made the changes I suggested, you can set the clock speed in `config.h`

**Pull request merged**